### PR TITLE
Add endpoint historical price from interval

### PR DESCRIPTION
**Pull request by laurentfrenette**

### CODE REVIEW

**Comment by laurentfrenette** [comment]
Close #85

File: src/bullets/data_source/data_source_fmp.py [hunk 1]
```diff
@@ -189,6 +220,7 @@ def _get_cached_cash_flow_statement(self, symbol: str, wanted_date: date):
 
     def _store_price_points(self, symbol: str, start_date: datetime.date, end_date: datetime.date = None,
                             limit: int = 1000):
+        print("I am called with " + str(start_date) + " to " + str(end_date))
```
Comment (SamiMammouche) [inline src/bullets/data_source/data_source_fmp.py]: I think you forgot to remove this line after debugging

**Comment by laurentfrenette** [comment]
@Raphael-Leb Any reasons was the PR was closed? Or was that just a missclick?

File: src/bullets/data_source/data_source_fmp.py (hunk 1 above)
Comment (laurentfrenette) [reply src/bullets/data_source/data_source_fmp.py]: Indeed, thanks!

Comment (laurentfrenette) [reply src/bullets/data_source/data_source_fmp.py]: Fixed

File: src/bullets/data_source/recorded_data.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+from datetime import datetime
+
```
Comment (SamiMammouche) [inline src/bullets/data_source/recorded_data.py]: Sorry for nitpicking, but there shouldn't be a blank line here

Comment (laurentfrenette) [reply src/bullets/data_source/recorded_data.py]: No prob, fixed :)

**Review by SamiMammouche — APPROVED** [review]

**Pull request merged**